README.md

```diff
@@ -200,14 +200,33 @@ repetitive tasks could be scripted.
 
 ### Communicate
 
-#### Don't be a lonesome cowboy/cowgirl coder.
-- Ask for feedback early and regularly. Embrace critique.
-- Embrace feedback in the code review process.
-- Use the [30/90% principle](https://42floors.com/blog/startups/thirty-percent-feedback) when giving feedback.
-- Seek opportunities to work with others and learn from them. There are lots of smart colleagues around you :).
-- *Consult!* There's a great tradition in medicine, where a doctor, no matter how experienced, will ask a colleague for a _consult_: an opportunity to hear someone else's take on a problem and/or a solution, and maybe open up a different perspective. Feel free to ask for consults, and to give them. Asking for one doesn't commit you to do anything more than listen to the answer. Being asked doesn't mean you have to solve someone else's problem. It's an opportunity to share.
+#### 👥 Seek Feedback
+
+> In medicine there is a tradition, where 👩🏾‍⚕️ doctors, no matter how
+> experienced, ask colleagues for a _consult_: an opportunity to hear another
+> perspective on how to resolve a challenge.
+
+_Mobify has blatantly stole this great tradition._
+
+**Actively seek feedback and incorporate it into your work.**
+
+Before writing code, discuss solutions with your peers.
```

Comment:
id specify something like "Before writing code for large changes or entirely new features/libraries/services".

I don't think people should discuss solutions for all the code they write.